web/app.js

```diff
@@ -2280,18 +2280,18 @@ if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
   };
 }
 
-function onPageRender({ pageNumber }) {
+function onPageRender({ pageNumber, source }) {
   // If the page is (the most) visible when it starts rendering,
   // ensure that the page number input loading indicator is displayed.
-  if (pageNumber === this.page) {
+  if (pageNumber === this.page && !source.detailView) {
```

Comment:
Note to myself: I need to change this, as the logic was for when the detail canvas was rendered _before_ the background canvas.